containers/run_hydra.sh

```diff
@@ -209,7 +209,11 @@ if [ ${#HC_KNOWN_HOSTS[@]} -ne 0 ] && [ ! -f "$KNOWN_HOSTS" ]; then
         h_name="${host%%:*}"
         h_port="${host##*:}"
         echo "Scanning host key for ${h_name}:${h_port}"
-        ssh-keyscan -H -p "$h_port" "$h_name" >> "$KNOWN_HOSTS" 2>/dev/null
+        if ! ssh-keyscan -H -p "$h_port" "$h_name" >> "$KNOWN_HOSTS" 2>/dev/null
+        then
```

Comment:
Bash script guidelines say that then should be on the same line with if
Also file otherwise complies already with this rule.

Reply:
Right. I were editing this in a relative narrow window and erroneously prioritized usual line length practices over the rule of having "then" on the same line. Will fix to match guidelines of the project.